python/src/usecases/logistic_model.py

```diff
@@ -22,6 +23,9 @@ class LogisticModel:
     b : Constant
         Parameter of the model b=1.5887e-10
 
+    nbdates : Constant
+        The number of years in the dataset.
+
     distY0 : `Normal` distribution
              ot.Normal(y0, 0.1 * y0)
 
```

Comment:
Below this line, several attribute descriptions could be edited to create links to the relevant classes, but they are out of reach of Github suggestions.

Reply:
I formatted the three use cases Flood, Chaboche and Logistic using appropriate back quotes or Sphinx links where possible.